wpilibcExamples/src/main/cpp/examples/GearsBot/cpp/subsystems/DriveTrain.cpp

```diff
@@ -21,10 +22,8 @@ DriveTrain::DriveTrain() {
   m_rightEncoder.SetDistancePerPulse(0.042);
 #else
   // Circumference in ft = 4in/12(in/ft)*PI
-  m_leftEncoder.SetDistancePerPulse(static_cast<double>(4.0 / 12.0 * M_PI) /
-                                    360.0);
-  m_rightEncoder.SetDistancePerPulse(static_cast<double>(4.0 / 12.0 * M_PI) /
-                                     360.0);
+  m_leftEncoder.SetDistancePerPulse(4.0 / 12.0 * wpi::math::pi / 360.0);
+  m_rightEncoder.SetDistancePerPulse(4.0 / 12.0 * wpi::math::pi / 360.0);
```

Comment:
Should we also be using units here?

Reply:
The Encoder class isn't unit-aware, so we can't. We'd have to make it a template class.

Reply:
I was thinking more of the conversion from in to ft.